ui/src/groups/GroupAdmin/GroupInfoEditor.tsx

```diff
@@ -163,7 +163,11 @@ export default function GroupInfoEditor({ title }: ViewProps) {
           >
             Delete {group?.meta.title}
           </button>
-          <DialogContent containerClass="max-w-[420px]">
+          <DialogContent
+            onInteractOutside={() => setDeleteDialogOpen(false)}
```

Comment:
You shouldn't have to do this or the onclick on `DialogClose`. you can add an onOpenChange to the `Dialog` component that does this for all these cases

Reply:
This seems to be the only instance it was used in, pushing the fix now.